specification/app/resource-manager/Microsoft.App/preview/2024-08-02-preview/CommonDefinitions.json

```diff
@@ -1614,6 +1614,208 @@
         }
       }
     },
+    "ManagedHttpRoutePatch": {
+      "description": "A managed http route to update",
+      "type": "object",
+      "properties": {
+        "tags": {
+          "type": "object",
+          "additionalProperties": {
```

Comment:
>additionalProperties [](http://example.com/codeflow?start=11&length=20)

ARM is trying to reduce instances of unschematized APIs to make it easier for consumers of those APIs to understand them and use them reliably, not to mention provide better support in the various SDKs.

Please make this an array or provide an explanation for why you need this.

Reply:
this is the PATCH response. Why would this need to be an array ? Could you give me an example spec of using arrays to for PATCH request/response ?

Reply:
@razvanbadea-msft can we resolve this conversation? this pattern is similar to all other tag props in the CommonDefintions spec

Reply:
This is for PATCH response and i see other resources using this pattern.

Managed Cert
https://github.com/Azure/azure-rest-api-specs/blob/508abd045ddc850c620270dd361e0ad589b25fd8/specification/app/resource-manager/Microsoft.App/preview/2024-08-02-preview/CommonDefinitions.json#L1608

Cert
https://github.com/Azure/azure-rest-api-specs/blob/508abd045ddc850c620270dd361e0ad589b25fd8/specification/app/resource-manager/Microsoft.App/preview/2024-08-02-preview/CommonDefinitions.json#L1534


Can we keep this pattern ?